### DISCUSSION TITLE
what is the elementId in  Html5Qrcode constructor ?

### DISCUSSION THREAD

**Discussion by ghost** [mebjas/html5-qrcode#746] [Q&A]
I'm trying to use  Html5Qrcode since I have an UI .
While initialising  Html5Qrcode , it is requried to pass in a elementID . 
Now, what is the element id here ? 
My camera preview is rendered on a canvas , so should I pass the canvas ID ?
Also is there a detailed doc on how to Html5Qrcode ?

**Comment by MarkerB** [comment]
There are docs on how to use Html5Qrcode, including this one that shows a specific example of the &lt;div&gt; container for the video preview:

[https://scanapp.org/html5-qrcode-docs/docs/intro#setup-a-target-html-container](https://scanapp.org/html5-qrcode-docs/docs/intro#setup-a-target-html-container)
